scrapy_zyte_api/providers.py

```diff
@@ -47,7 +63,14 @@ async def __call__(
             return results
 
         html_requested = BrowserResponse in to_provide or BrowserHtml in to_provide
-        item_keywords = {Product: "product"}
+        item_keywords = {
+            Product: "product",
+            ProductList: "productList",
+            ProductNavigation: "productNavigation",
+            Article: "article",
+            ArticleList: "articleList",
+            ArticleNavigation: "articleNavigation",
```

Comment:
I haven't added tests for this, because it doesn't seem they add much, as compared to the existing Product test.